pages/test_analytics/ruby_collectors.md.erb

```diff
@@ -99,6 +99,20 @@ Once you're done, in your Test Analytics dashboard, you'll see analytics of test
 
 If you don't see branch names, build numbers, or commit hashes in the Test Analytics UI, then see [CI Environments](/docs/test-analytics/ci-environments) to learn more about exporting your environment to the minitest collector.
 
+## Adding annotation spans
+
+This gem allows adding custom annotations to the span data sent to Buildkite using the [.annotate](https://github.com/buildkite/test-collector-ruby/blob/d9fe11341e4aa470e766febee38124b644572360/lib/buildkite/test_collector.rb#L64) method. For example:
+
+```ruby
+Buildkite::TestCollector.annotate("Rendered OTP Screen")
+```
+
+This would appear like so:
+
+<%= image "annotation-span.png", width: 512, height: 359, alt: "Screenshot of the span timeline including the 'Rendered OTP Screen' annotation." %>
```

Comment:
Could we take a `@2x` screenshot from a retina/high dpi screen? Please replace `annotation-span.png` with an image that's double the dimension. This improves the image quality 🖼️ ✨ for devices like Macbooks or mobile devices.

After replacing the image file you can then implement the responsive image with:

```suggestion
<%= image "annotation-span.png", width: 1024/2, height: 718/2, alt: "Screenshot of the span timeline including the 'Rendered OTP Screen' annotation." %>
```

Reply:
I had a go at taking a screenshot from the app. What do folks think about this one? 

<img width="1024" alt="annotation-span" src="https://user-images.githubusercontent.com/7202667/193493948-d1e54488-32bc-42f6-bfc5-ccbbf55d9a2b.png">

Reply:
I like it. Give me a couple of minutes and I'll add it in.